test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -403,6 +403,31 @@ UCS_TEST_P(test_ucp_tag_offload, eager_multi_probe,
     request_wait(rreq);
 }
 
+UCS_TEST_P(test_ucp_tag_offload, eager_multi_recv,
+           "RNDV_THRESH=inf", "TM_THRESH=0")
+{
+    activate_offload(sender());
+
+    size_t length = ucp_ep_config(sender().ep())->tag.rndv.am_thresh.remote - 1;
+    ucp_tag_t tag = 0x11;
+    std::vector<uint8_t> sendbuf(length);
+
+    ucp_request_param_t param = {};
+    ucs_status_ptr_t sreq     = ucp_tag_send_nbx(sender().ep(), sendbuf.data(),
+                                                 sendbuf.size(), tag, &param);
+
+    // Tweak progress several times to make sure the first fragment
+    // (but not all!) arrives
```

Comment:
1. does it mean need to run test several times to repro? if yes pls add to comment
2. what is the symptom if the test fails? receive request is not completed? add to the comment

Reply:
added comments